client/Makefile

```diff
@@ -25,20 +25,21 @@ vpath %.dic dictionaries
 OBJDIR = obj
 
 ifeq ($(USE_BREW),1)
-	ifdef MACPORTS_PREFIX
-		INCLUDES += -I$(MACPORTS_PREFIX)/include
-		LDLIBS += -L$(MACPORTS_PREFIX)/lib
-		PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/lib/pkgconfig
-		PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/libexec/qt/lib/pkgconfig
-		PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/libexec/qt5/lib/pkgconfig
-	else
-		INCLUDES += -I$(BREW_PREFIX)/include
-		LDLIBS += -L$(BREW_PREFIX)/lib
-		PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(BREW_PREFIX)/opt/qt/lib/pkgconfig
-		PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(BREW_PREFIX)/opt/qt5/lib/pkgconfig
-	endif
+	INCLUDES += -I$(BREW_PREFIX)/include
+	LDLIBS += -L$(BREW_PREFIX)/lib
+	PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(BREW_PREFIX)/opt/qt/lib/pkgconfig
+	PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(BREW_PREFIX)/opt/qt5/lib/pkgconfig
 endif
 
+ifdef ($(USE_MACPORTS),1)
+	INCLUDES += -I$(MACPORTS_PREFIX)/include
+	LDLIBS += -L$(MACPORTS_PREFIX)/lib
+	PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/lib/pkgconfig
```

Comment:
no need for the 
```
PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/libexec/qt/lib/pkgconfig
PKG_CONFIG_ENV := PKG_CONFIG_PATH=$(MACPORTS_PREFIX)/libexec/qt5/lib/pkgconfig
```
that were there previously ?

Reply:
BTW CI still failing..

Reply:
yeah the macports maintainers notified me that having QT5 enabled in their Portfile will append them automatically